tests/tests/utils/menu.ts

```diff
@@ -11,16 +11,20 @@ class Menu {
         await page.getByTestId('map-menu').click({timeout: 30_000});
         await page.getByRole('button', { name: 'Map editor' }).click();
         await expect(page.getByRole('button', { name: 'Map editor' })).toBeHidden();
-        // await expect(await page.getByRole('button', {name: 'toggle-map-editor'}).first()).toHaveClass(/border-top-light/);
+        // await expect(await page.getByRole('button', {name: 'toggle-map-editor'}).first()).toHaveClass(/bor²der-top-light/);
     }
 
-    async openMenu(page: Page, isMobile: boolean) {
-        if(isMobile){
-            await expect(page.locator('button#burgerIcon')).toBeVisible();
-            const mobileMenuVisible = await page.locator('button#burgerIcon img.tw-rotate-0').isVisible();
-            if(mobileMenuVisible){
-                await page.click('button#burgerIcon');
-            }
+    async openBurgerMenu(page: Page) {
+        await page.getByTestId('burger-menu').click();
+        await expect(page.getByText('Change your status')).toBeVisible();
+    }
+
+    async openMenu(page: Page) {
+        // 'button#burgerIcon' do not exist in the new graphic version !!
+        await expect(page.locator('button#burgerIcon')).toBeVisible();
```

Comment:
"burgerIcon" does not exist anymore. You should remove the "if" block code instead of using it.